### PR TITLE
New Template: cisco_nxos_show_ip_interface

### PR DESCRIPTION
**Pull request by network-dave**
##### ISSUE TYPE

 - New Template Pull Request

##### COMPONENT

Cisco NXOS - show ip interface

##### SUMMARY

- Secondary IP addresses are stored as a list object.
- Multicast groups are stored in one single string, they can't be split in multiple variables with the actual version of TextFSM (this might be the case later on)

### CODE REVIEW

File: templates/cisco_nxos_show_ip_interface.textfsm [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  ^\s+Multicast\s+bytes\s+:\s+${MULTICAST_BYTES_SENT}\/${MULTICAST_BYTES_RECEIVED}\/${MULTICAST_BYTES_FORWARDED}\/${MULTICAST_BYTES_ORIGINATED}\/${MULTICAST_BYTES_CONSUMED}\s*$$
+  ^\s+Broadcast\s+packets\s+:\s+${BROADCAST_PACKETS_SENT}\/${BROADCAST_PACKETS_RECEIVED}\/${BROADCAST_PACKETS_FORWARDED}\/${BROADCAST_PACKETS_ORIGINATED}\/${BROADCAST_PACKETS_CONSUMED}\s*$$
+  ^\s+Broadcast\s+bytes\s+:\s+${BROADCAST_BYTES_SENT}\/${BROADCAST_BYTES_RECEIVED}\/${BROADCAST_BYTES_FORWARDED}\/${BROADCAST_BYTES_ORIGINATED}\/${BROADCAST_BYTES_CONSUMED}\s*$$
+  ^\s+Labeled\s+packets\s+:\s+${LABELED_PACKETS_SENT}\/${LABELED_PACKETS_RECEIVED}\/${LABELED_PACKETS_FORWARDED}\/${LABELED_PACKETS_ORIGINATED}\/${LABELED_PACKETS_CONSUMED}\s*$$
+  ^\s+Labeled\s+bytes\s+:\s+${LABELED_BYTES_SENT}\/${LABELED_BYTES_RECEIVED}\/${LABELED_BYTES_FORWARDED}\/${LABELED_BYTES_ORIGINATED}\/${LABELED_BYTES_CONSUMED}\s*$$
+  ^\s+WCCP\s+Redirect\s+outbound:\s+${WCCP_REDIRECT_OUTBOUND}\s*$$
+  ^\s+WCCP\s+Redirect\s+inbound:\s+${WCCP_REDIRECT_INBOUND}\s*$$
+  ^\s+WCCP\s+Redirect\s+exclude:\s+${WCCP_REDIRECT_EXCLUDE}\s*$$ -> Record
+  ^. -> Error
```
Comment (FragmentedPacket) [inline templates/cisco_nxos_show_ip_interface.textfsm:105]:
```suggestion
  ^\s*$$
  ^. -> Error
```

File: templates/cisco_nxos_show_ip_interface.textfsm [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+Value LABELED_BYTES_RECEIVED (\d+)
+Value LABELED_BYTES_FORWARDED (\d+)
+Value LABELED_BYTES_ORIGINATED (\d+)
+Value LABELED_BYTES_CONSUMED (\d+)
+Value WCCP_REDIRECT_OUTBOUND (\S+)
+Value WCCP_REDIRECT_INBOUND (\S+)
+Value WCCP_REDIRECT_EXCLUDE (\S+)
+
+Start
+  ^IP\s+Interface\s+Status\s+for\s+VRF\s+\"${VRF_NAME}\"\s*$$
```
Comment (FragmentedPacket) [inline templates/cisco_nxos_show_ip_interface.textfsm:71]:
```suggestion
Start
  ^IP\s+Interface\s+Status\s+for\s+VRF\s+\"${VRF_NAME}\"\s*$$
  ^\S+, -> Continue.Record
```

Remove the record on WCCP line.

Comment (network-dave) [reply templates/cisco_nxos_show_ip_interface.textfsm:71]: Wouldn't this also work ?

^${INTERFACE},.*protocol-${PROTOCOL_STATUS}\/link-${LINK_STATUS}\/admin-${ADMIN_STATUS},.*$$ -> Continue.Record

Comment (network-dave) [reply templates/cisco_nxos_show_ip_interface.textfsm:71]: Unfortunately this doesn't work, some interfaces get the wrong VRF value

File: templates/cisco_nxos_show_ip_interface.textfsm (hunk 1 above)
Comment (network-dave) [reply templates/cisco_nxos_show_ip_interface.textfsm:105]: Done

**Pull request merged**